### PR TITLE
CORE-11523: Boot parameters for REST worker

### PR DESCRIPTION
**Pull request by vkolomeyko**
This is a non-breaking change.

### CODE REVIEW

File: data/config-schema/src/main/java/net/corda/schema/configuration/BootConfig.java [hunk 1]
```diff
@@ -26,4 +27,9 @@ private BootConfig() {
     public static final String BOOT_DIR = "dir";
     public static final String BOOT_WORKSPACE_DIR = BOOT_DIR + ".workspace";
     public static final String BOOT_TMP_DIR = BOOT_DIR + ".tmp";
+
+    public static final String BOOT_REST = "rest";
+    public static final String BOOT_REST_PARAMS = BOOT_REST + ".params";
```
Comment (driessamyn) [inline data/config-schema/src/main/java/net/corda/schema/configuration/BootConfig.java:32]: Does it need `params`?
It seems maybe a bit un-necessary. Maybe `rest.tls.keystore...` is good enough?

Comment (vkolomeyko) [reply data/config-schema/src/main/java/net/corda/schema/configuration/BootConfig.java:32]: I am flexible either way. But I was trying to be consistent with DB, i.e.: `BOOT_DB_PARAMS` which is `db.params....`.

Either way command line argument will look like: `-rtls.keystore.file.path=...` so use of `params` is purely internal and not visible to the C5 operator using REST Worker images.

Comment (driessamyn) [reply data/config-schema/src/main/java/net/corda/schema/configuration/BootConfig.java:32]: Ah. Yes. Got it 👍

**Review by driessamyn — APPROVED** [review]
LGTM

**Pull request merged**